automation/steps/components/Button/ButtonSteps.ts

```diff
@@ -6,6 +6,10 @@ class ButtonSteps {
     public async clickButton<T extends BaseComponent>(type: { new(page: Page, identifier: string): T; }, identifier: string, parent?: Locator) {
         await (await driver.component(type, identifier, parent)).click();
     }
+
+    public async clickLastButton<T extends BaseComponent>(type: { new(page: Page, identifier: string): T; }, identifier: string, parent?: Locator) {
```

Comment:
have no see sense in this method, if you want to select some special table line and interact with it, that would be better , just example how to interact with table-rows (//td[contains(text(),'Oleg')]//ancestor::tr//td//a[@data-id="button-Delete"]).  THAT'S JUST EXAMPLE